packages/mdc-slider/component.ts

```diff
@@ -118,24 +118,8 @@ export class MDCSlider extends MDCComponent<MDCSliderFoundation> {
       },
       setTrackStyleProperty: (propertyName, value) => this.track_.style.setProperty(propertyName, value),
       setMarkerValue: (value) => this.pinValueMarker_.innerText = value.toLocaleString(),
-      appendTrackMarkers: (numMarkers) => {
-        const frag = document.createDocumentFragment();
-        for (let i = 0; i < numMarkers; i++) {
-          const marker = document.createElement('div');
-          marker.classList.add('mdc-slider__track-marker');
-          frag.appendChild(marker);
-        }
-        this.trackMarkerContainer_.appendChild(frag);
-      },
-      removeTrackMarkers: () => {
-        while (this.trackMarkerContainer_.firstChild) {
-          this.trackMarkerContainer_.removeChild(this.trackMarkerContainer_.firstChild);
-        }
-      },
-      setLastTrackMarkersStyleProperty: (propertyName, value) => {
```

Comment:
Please remove all of these adapter methods from the README.md. Also add the new `setTrackMarkers` method.